lib/rules/spaced-comment.js

```diff
@@ -219,13 +246,16 @@ module.exports = {
          * comments.
          */
         var config = context.options[1] || {};
+        var balanced = config.block && config.block.balanced || false;
```

Comment:
I think this line can just be `var balanced = config.block && config.block.balanced`